tests/test_h2_client.c

```diff
@@ -155,3 +155,22 @@ TEST_CASE(h2_client_ping_ack) {
     return s_tester_clean_up();
 }
 /* TODO: test that ping response is sent with higher priority than any other frame */
+
+TEST_CASE(h2_client_setting_ack) {
+    ASSERT_SUCCESS(s_tester_init(allocator, ctx));
+
+    /* Connection preface requires that SETTINGS be sent first (RFC-7540 3.5). */
+    ASSERT_SUCCESS(h2_fake_peer_send_connection_preface_default_settings(&s_tester.peer));
+    testing_channel_drain_queued_tasks(&s_tester.testing_channel);
+
+    /* Have the fake peer to run its decoder on what the client has written.
+     * The decoder will raise an error if it doesn't receive the "client connection preface string" first. */
```

Comment:
trivial: remove "The decoder will raise an error if it doesn't receive the "client connection preface string" first."
that's from the "connection preface" tests, we don't need to copy/paste it into every further test we write